base_field_validator/demo/ir_model_regex_demo.yml

```diff
@@ -0,0 +1,12 @@
+-
+  set regex for partner
+-
+  !python {model: ir.model}: |
+    self.write(cr, uid, [ref('base.model_res_partner')], {
+      'validator_line_ids': [(0,0,
+        {
+        'model_id': ref('base.model_res_partner'),
+        'field_id': ref('base.field_res_partner_email'),
+        'regex_id': ref('regex_mail'),
+        })],
```

Comment:
Although this is correct and I won't ask you to change it, you'll like to know that there's a much better syntax for creating records in YAML files. This could become:

```
- !record {model: ir.model, id: base.model_res_partner}:
    validator_line_ids:
      - model_id: base.model_res_partner
        field_id: base.field_res_partner_email
        regex_id: regex_mail
```

You can [see other examples](https://github.com/odoo/odoo/search?l=yaml&q=!record&utf8=%E2%9C%93) if you are interested.